Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -220,11 +221,19 @@ verifyRCFlow person merchantOpCityId imageExtraction rcNumber imageId dateOfRegi
         if isNothing dateOfRegistration && imageExtraction
           then Domain.Success
           else Domain.Skipped
+  config <- CQMSUC.findByMerchantOpCityId merchantOpCityId >>= fromMaybeM (MerchantServiceUsageConfigNotFound merchantOpCityId.getId)
```

Comment:
Why to fetch the config here. You can move it inside the `verifyRC` function inside `Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Tools/Verification.hs`

Reply:
done